src/analysis/special_functions/pow.lean

```diff
@@ -1248,6 +1255,44 @@ lemma rpow_le_rpow_of_exponent_ge {x : ℝ≥0} {y z : ℝ} (hx0 : 0 < x) (hx1 :
   x^y ≤ x^z :=
 real.rpow_le_rpow_of_exponent_ge hx0 hx1 hyz
 
+lemma le_rpow_inv_iff_of_neg {a b c : ℝ} (ha : 0 < a) (hb : 0 < b) (hc : c < 0) :
```

Comment:
Can you add the `_of_pos` versions of these lemmas too? Or do they already exist?